Tests/TestUtilities/TestUtils.cs

```diff
@@ -78,6 +78,25 @@ public static string EnsureAnalysisTargetsExists(TestContext testContext)
             return filePath;
         }
 
+        /// <summary>
+        /// Ensures the default properties file exists in the specified folder
+        /// </summary>
+        public static string EnsureDefaultPropertiesFileExists(string targetDir, TestContext testContext)
+        {
+            string filePath = Path.Combine(targetDir, SonarQube.Common.FilePropertyProvider.DefaultFileName);
+            if (File.Exists(filePath))
+            {
+                testContext.WriteLine("Default properties file already exists: {0}", filePath);
+            }
+            else
+            {
+                testContext.WriteLine("Extracting default properties file to {0}", filePath);
+                Directory.CreateDirectory(targetDir);
+                ExtractResourceToFile("TestUtilities.Embedded.SonarQube.Analysis.xml", filePath);
+            }
+            return filePath;
+        }
+
```

Comment:
Some tests need to be able to deploy the default properties file to a known directory. There were already similar methods that deployed the two targets files; this method uses the same pattern for the default properties file.